sdcm/utils/version_utils.py

```diff
@@ -192,6 +219,18 @@ def get_branch_version(url):
     return []
 
 
+def get_all_versions(url: str) -> set[str]:
+    repo_details = get_repository_details(url=url)
+    urls = get_scylla_urls_from_repository(repo_details=repo_details)
+
+    if repo_details.type == ScyllaFileType.DEBIAN:
+        return get_all_versions_from_debian_repository(urls=urls)
+    elif repo_details.type == ScyllaFileType.YUM:
+        return get_all_versions_from_centos_repository(urls=urls)
+    # To overcome on Pylint's "inconsistent-return-statements", a value must be returned
+    return set()
```

Comment:
is this a valid use-case? would it be usefull at least print a warning about it?
why not simply returning 'None'?

Reply:
there are only 2 options:
https://github.com/scylladb/scylla-cluster-tests/blob/728debeaa5b948f2e9776ced40ea15ae4034c8e4/sdcm/utils/version_utils.py#L70

and as the comment is saying, it's just to please pylint, hence no need to print a warning.